wallet-new/integration/Main.hs

```diff
@@ -67,35 +27,13 @@ main = do
     --     Try `cardano-integration-test --help' for more information.
     --
     -- See also: https://github.com/hspec/hspec/issues/135
-    withArgs [] . hspec $ deterministicTests wRef walletClient manager
-  where
-    orFail :: MonadFail m => Either String a -> m a
-    orFail =
-        either (fail . ("Error decoding X509 certificates: " <>)) return
-
-
-initialWalletState :: WalletClient IO -> IO WalletState
-initialWalletState wc = do
-    -- We will have single genesis wallet in intial state that was imported from launching script
-    _wallets <- fromResp $ getWallets wc
-    _accounts <- concat <$> for _wallets (fromResp . getAccounts wc . walId)
-    -- Lets set all wallet passwords for initial wallets (genesis) to default (emptyPassphrase)
-    let _lastAction       = NoOp
-        _walletsPass      = fromList $ map ((, V1 mempty) . walId) _wallets
-        _addresses        = concatMap accAddresses _accounts
-        -- TODO(akegalj): I am not sure does importing a genesis wallet (which we do prior launching integration tests) creates a transaction
-        -- If it does, we should add this transaction to the list
-        _transactions     = mempty
-        _actionsNum       = 0
-        _successActions   = mempty
-    pure $ WalletState {..}
-  where
-    fromResp = (either throwM (pure . wrData) =<<)
+    withArgs [] . hspec $ noneDeterministicTests walletClient manager
 
-deterministicTests :: WalletRef -> WalletClient IO -> Manager -> Spec
-deterministicTests wref wc manager = do
+noneDeterministicTests :: WalletClient IO -> Manager -> Spec
+noneDeterministicTests wc manager = do
```

Comment:
These tests are deterministic; in that the same spec tree runs for each test. In comparison to the non-deterministic tests, which do a random walk through the state space. Why change the name?